l10n_es_verifactu/models/verifactu_send_queue.py

```diff
@@ -57,12 +57,30 @@ class VerifactuSendQueue(models.Model):
         help="True if a correction has been made to the document, and it must be resent."
     )
     company_id = fields.Many2one("res.company", required=True)
-    response_ids = fields.One2many(
+    response_line_ids = fields.One2many(
         "verifactu.send.response.line",
         "send_queue_id",
         string="Responses",
         help="Responses from Verifactu after sending the documents.",
     )
+    last_error_code = fields.Char(compute="_compute_last_error_code", store=True)
+
+    @api.depends("response_line_ids", "response_line_ids.send_state")
+    def _compute_send_state(self):
+        for rec in self:
+            rec.send_state = "not_sent"
+            last_response = rec.response_line_ids and rec.response_line_ids[0]
+            if last_response:
+                rec.send_state = last_response.send_state
```

Comment:
esto podríamos resumirlo así
```suggestion
            rec.send_state = rec.response_line_ids[:1].send_state or "not_sent"
```